httpx/_api.py

```diff
@@ -34,7 +34,7 @@ def request(
     auth: AuthTypes = None,
     proxies: ProxiesTypes = None,
     timeout: TimeoutTypes = DEFAULT_TIMEOUT_CONFIG,
-    allow_redirects: bool = True,
+    follow_redirects: bool = False,
```

Comment:
I assume we don't consider a soft deprecation (with a warning) to be worth it here, given the amount of code that would require?